gammapy/background/energy_offset_array.py

```diff
@@ -0,0 +1,105 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Please add this boilerplate line:
```
from __future__ import absolute_import, division, print_function, unicode_literals
```
This is to make simultaneous Python 2 / 3 codebase easier and we have it in every file.